src/modules/Builder/features/ActivitySettings/ScoresAndReports/ScoreContent/ScoreContent.utils.ts

```diff
@@ -136,20 +154,27 @@ const updateMessage = (
 
 export const updateMessagesWithVariable = (
   setValue: UseFormSetValue<FieldValues>,
```

Comment:
could you please use object parameters and move according type to ScoreContent.types.ts?